var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py

```diff
@@ -20,7 +20,7 @@ class NeurodamusHippocampus(NeurodamusModel):
     # IMPORTANT: Register versions (only) here to make them stable
     # Final version name is combined e.g. "1.0-3.0.1"
     model_core_dep_v = (
-        ('1.4.1', '3.2.0'),
+        ('1.4.1', '3.2.2'),
```

Comment:
Should we keep the old versions as well? @ferdonline @pramodk

Reply:
It's in archive. So I think its ok to rid of it and avoid confusion.